demo/App.tsx

```diff
@@ -160,10 +175,18 @@ const App = () => {
                 color="green"
                 testID="addCustomDataBtn"
                 accessibilityLabel="addCustomDataBtn"
-                onPress={() =>
-                  RaygunClient.addCustomData({
-                    [Date.now()]: `Random: ${Math.random() * 1000}`
-                  })
+                onPress={() => {
+                  const customData1: CustomData = {"Key_1": "Value"};
+                  const customData2: CustomData = {"Key_2": 401};
+                  const customData3: CustomData = {"Key_3": ["Value", "Another Value"] };
+                  const customData4: CustomData = {"Key_4": [42, 65] };
+                  const customData5: CustomData = {"Key_5": customData4};
+                  RaygunClient.addCustomData(customData1);
+                  RaygunClient.addCustomData(customData2);
+                  RaygunClient.addCustomData(customData3);
+                  RaygunClient.addCustomData(customData4);
+                  RaygunClient.addCustomData(customData5);
+                }
                 }
                 title="Add Random Custom Data"
```

Comment:
This is no longer adding random custom data, so the title will need updating

Reply:
DONE